introduction.md

```diff
@@ -3,3 +3,6 @@
 I'm David, @deibiz, http://www.davidgg.es.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?